cmd/server/config.go

```diff
@@ -123,8 +135,9 @@ func DefaultConfig() *server.Config {
 			CgroupManager:   cgroupManager,
 		},
 		ImageConfig: server.ImageConfig{
-			Pause:    pausePath,
-			ImageDir: filepath.Join(ocidRoot, "store"),
+			DefaultTransport: defaultTransport,
+			PauseImage:       pauseImage,
+			PauseCommand:     pauseCommand,
```

Comment:
If there's a config blob and it can be decoded as an image-spec v1 image, yes.  If that's not every image, we'll probably have to have some default.